src/foam/nanos/u2/navigation/SignIn.js

```diff
@@ -27,7 +27,7 @@ foam.CLASS({
       hidden: true
     },
     {
-      class: 'EMail',
+      class: 'String',
```

Comment:
Why the change back to String?

Reply:
I remembered someone had mentioned not wanting to show validation when users sign up. If we set class to EMail, it will display validation text so I reverted it back to how it had been.

Reply:
Could you leave it as Email but override the view for it in the sign in form instead?

Reply:
I think this is the reason why someone who wrote this code set class to String in the first place.

Reply:
Yes, why?

Reply:
I think you wouldn't usually have validation text display on login page (or at least I think it was the writer's intention). @TW80000 I will try your approach.